packages/oscal-react-library/src/components/OSCALBackMatter.test.tsx

```diff
@@ -9,56 +8,50 @@ import {
   exampleBackMatterWithoutMediaType,
 } from "../test-data/BackMatterData";
 
-function backMatterRenderer() {
-  render(<OSCALBackMatter backMatter={backMatterTestData} parentUrl={parentUrlTestData} />);
-}
-
-export default function testOSCALBackMatter(parentElementName, renderer) {
-  test(`${parentElementName} displays resource title`, () => {
-    renderer();
+describe("OSCAL Backmatter", () => {
+  test("displays resource title", () => {
+    render(<OSCALBackMatter backMatter={backMatterTestData} parentUrl={parentUrlTestData} />);
```

Comment:
This Backmatter test data is duplicated throughout the tests. Is there any reason to avoid having a renderer function?

Reply:
I agree 100%. This is a small enough difference that I think it's valuable to just make the call. Additional indirection is not useful here and is actually less helpful for debugging.

I am very much opposed to these styles of "renderer()" functions.

What is the alternative? Calling `renderer()` over and over? It's still one line repeated many times. Not sure the juice is worth the squeeze.

In fact, I'd argue that our tests _should not_ be using the same data and the same parameters _every_ time; and that would discourage some magic function that solves the problem of repetition. We should solve it by it not existing in the first place